vanilla-framework/vanilla-framework
# 770 Split out and rename visibility util

## Done

It was raised during Vanilla sprint that this util is confusing as it's named after a different CSS property than the one it implemented.

Deprecated `.u-hidden` / `.u-visible` in favour of `.u-show` / `.u-hide`.

Also: I removed an errant underscore from the `animation-settings.md` file

## QA

- Pull code
- Run `gulp build` - there should be no errors
- Run `gulp test` - there should be no errors
- Run `gulp jekyll`
- Check show and hide examples


## Details

Fixes #770
